packages/react-components/react-tags/src/components/Tag/renderTag.tsx

```diff
@@ -23,9 +23,7 @@ export const renderTag_unstable = (state: TagState, contextValues: TagContextVal
             </AvatarContextProvider>
           )}
           {slots.icon && <slots.icon {...slotProps.icon} />}
-          {slots.primaryText && (
```

Comment:
if `primaryText` is optional in the render method, then would be good to add a `?` to it's declaration on `TagSlots`, same goes for `content`, `media`, `icon` and `secondaryText`

Reply:
If it's a required slot, then there's no need for this verification, and also it would be required to add `NonNullable` in the type declaration on `TagSlots`

Reply:
Looking at the design spec this should be a required slot always, since the children are directly inserted there